johnsonandjohnson/Bodiless-JS
# feat(core-ui, layouts, layouts-ui, ui): Update Flexbox Fly-Out Panel UI

## Changes

- Updated Component Picker width and height.
- Component Picker opens in the middle of the page and goes off the screen issue is fixed.
- Fixes issue with components preview were cut in half on the first ComponentPicker open.

## Test Instructions

- Run `npm run setup` first. It will update auto-generated `.css` files.
- Run `npm run start` and go to `/flexbox` page.
- Open Component picker and make sure `.rc-tooltip-content` div has right demensions. ( 714px x 744px ) 

- Add more content to the `Bodiless-JS/examples/test-site/src/data/pages/flexbox/index.tsx` so that you can scroll the page.
- Run `npm run start` and go to `/flexbox` page.
- Activate + icon by clicking on any flexbox areas and scroll page down.
- Try to open Component Picker -- it should stick to the top.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/54
